target/Dockerfile

```diff
@@ -13,4 +16,27 @@ RUN chown -R root:root /root && \
 
 COPY index.html /var/www/localhost/htdocs/
 COPY aica-nginx.conf /etc/nginx/http.d/default.conf
-COPY docker_entrypoint.sh /usr/local/bin/
\ No newline at end of file
+COPY docker_entrypoint.sh /usr/local/bin/
+COPY clamonacc.sh /usr/local/bin
+COPY test-files /opt/test-files
+
+RUN sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/clamd.conf && \
+    sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/freshclam.conf
+
+# this is to bootstrap the virus definitions, as trying to start clamd without
+# any just causes the process to die, upon the container starting it updates
+# these immediately again
+RUN freshclam
+RUN addgroup -S appuser && adduser -D -s /bin/bash appuser -G appuser 
+
+# The malware.zip.malz is a zip archive that is not in the repo that is used to generate
+# clamav logs. Please remove a line containing that file name if you see it.
+RUN mkdir /run/clamav && mkdir /root/quarantine && \
+    chown root:root /run/clamav && chown root:root /root/quarantine && \
+    chown appuser:appuser /opt/test-files/malware.zip.malz /opt/test-files/eicar.com && \
+    chmod +x /usr/local/bin/docker_entrypoint.sh /usr/local/bin/clamonacc.sh
+
+COPY etc /etc/
+
+# Only uncomment when using the build-docker.sh script
```

Comment:
Instead of trying to run all of the docker containers at once all of the time, I had put this in to do some unit testing, also because I usually do my own Docker stuff using a build script instead of docker-compose. This has been removed appropriately.

Reply:
Sorry, one more thing here - can we copy specifically what is needed in /etc instead of the entire etc directory?